libs/commands/version/src/lib/create-release.ts

```diff
@@ -17,20 +17,21 @@ export function createReleaseClient(type: "github" | "gitlab") {
 export function createRelease(
   client: ReturnType<typeof createReleaseClient>,
   { tags, releaseNotes }: { tags: string[]; releaseNotes: { name: string; notes: string }[] },
-  { gitRemote, execOpts }: { gitRemote: string; execOpts: ExecOptions }
+  { gitRemote, execOpts }: { gitRemote: string; execOpts: ExecOptions },
+  separator: string
```

Comment:
I don't think it's a good idea to add yet another argument, I think it might be better to include it in either 2nd or 3rd argument, this makes it easier to extend later if other arguments are to be added in the future

Reply:
looks good